### PR TITLE
Move sandbox into its own package

### PR DESCRIPTION
**Pull request by 14rcole**
If sandbox is in the same package as server, there will be a circular dependency when
kpod create is implemented

### CODE REVIEW

File: server/sandbox/sandbox.go [hunk 1]
```diff
@@ -72,9 +72,10 @@ func isSymbolicLink(path string) (bool, error) {
 	return fi.Mode()&os.ModeSymlink == os.ModeSymlink, nil
 }
 
-func netNsGet(nspath, name string) (*sandboxNetNs, error) {
+// Gets the NetNs associated with the given nspath and name
+func NetNsGet(nspath, name string) (*NetNs, error) {
 	if err := ns.IsNSorErr(nspath); err != nil {
-		return nil, errSandboxClosedNetNS
+		return nil, ErrSandboxClosedNetNS
```
Comment (rhatdan) [inline server/sandbox/sandbox.go]: Should error change to ErrClosedNetNS?

File: server/sandbox/sandbox.go [hunk 2]
```diff
@@ -153,20 +155,20 @@ type Sandbox struct {
...
-	podInfraCommand = "/pause"
+	DefaultShmSize  = 64 * 1024 * 1024
+	NsRunDir        = "/var/run/netns"
+	PodInfraCommand = "/pause"
 )
 
 var (
-	errSandboxIDEmpty     = errors.New("PodSandboxId should not be empty")
-	errSandboxClosedNetNS = errors.New("PodSandbox networking namespace is closed")
+	ErrSandboxIDEmpty     = errors.New("PodSandboxId should not be empty")
```
Comment (rhatdan) [inline server/sandbox/sandbox.go]: Should you remove the Sandbox?

Comment (14rcole) [inline server/sandbox/sandbox.go]: Yes, good call

File: server/sandbox/sandbox.go [hunk 3]
```diff
@@ -1,4 +1,4 @@
-package server
+package sandbox
```
Comment (mrunalp) [inline server/sandbox/sandbox.go]: Why not under libkpod?

Comment (mheon) [inline server/sandbox/sandbox.go]: @mrunalp Probably need to be a subpackage there, too (if we move kpod state out into a subpackage, we'll end up with another import cycle). `libkpod/sandbox` would be fine with me.

Comment (14rcole) [inline server/sandbox/sandbox.go]: I was under the impression that libkpod would only handle non-kubernetes related things for the time being, and that pods may be added in later.  If I were to add it to libkpod, should I add it as a subpackage or within the package itself?

Comment (mrunalp) [inline server/sandbox/sandbox.go]: We still need Sandboxes per the discussion last week to support Clear Containers well. 
libkpod, preferably, and a sub-package, if needed, to break cycles is fine with me.

Comment (mheon) [inline server/sandbox/sandbox.go]: @14rcole We're going to be leaving Sandboxes in the API, though we eventually want to refactor them to be less Kubernetes specific. `libkpod` is fine place, though it should live in a subpackage.

**Comment by mrunalp** [comment]
LGTM

**Pull request merged**